### PR TITLE
Tesla Nerf

### PR DESCRIPTION
**Pull request by Iamgoofball**
It no longer bounces shocks nearly as far., and can be heard 2 screens away now.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Feature freeze sorry

**Comment by Iamgoofball** [comment]
fuck

**Comment by Aranclanos** [comment]
oh baby

**Comment by Anonus** [comment]
chill out

**Comment by phil235** [comment]
I think we should make an exception to the freeze. Tesla badly needs a nerf and this PR only adjusts some numbers so it has no risk of creating new bugs.

**Comment by Iamgoofball** [comment]
Worst case scenario, tesla becomes underpowered, which is no biggie.

**Comment by Iamgoofball** [comment]
fuck it people are arguing too much over this shit and the muh freeze crap, maintainers can re-open when they want to merge be it this month or next month

**Comment by borntohonk** [comment]
do it

**Comment by Iamgoofball** [comment]
ooc threatened to murder me if I didn't reopen this

**Comment by Jalleo** [comment]
ROMANES EUNT DOMUS

**Comment by rev-benji** [comment]
Please merge this, tesla is very broken at the moment. Like phil said this isn't going to fuck anything up.  If you've played a round where the tesla gets released and it hangs around the station, you know how stupid deadly it is. I really don't want another month of GG NO RE

**Comment by Fox-McCloud** [comment]
still think the amoutn of power it generates needs to be more than cut in half--not to mention I still think a limit to the number of orbits is in order.

**Comment by Iamgoofball** [comment]
@Fox-McCloud orbital limit and power generation will not be touched until something is added that makes the amount of power beyond the bare minimum to keep the station running matters

**Comment by duncathan** [comment]
Tesla generates exactly as much power as the sing: enough.

**Comment by Fox-McCloud** [comment]
the tesla, with just its base power generation produces 3.5 million watts of power, which is already 50% more than the singulo--the orbiting balls could produce 0 additional power and it would still be MORE than enough to power the entire station and still wreck powersinks faster than the average singulo.

**Comment by Iamgoofball** [comment]
Wait, powersinks die from excessive power input?

That sounds like a bad feature, kinda kills the point of the powersink.

File: code/game/objects/objs.dm [hunk 1]
```diff
@@ -194,7 +194,7 @@
 
 /obj/proc/tesla_act(var/power)
 	being_shocked = 1
-	var/power_bounced = power / 1.3
-	tesla_zap(src, 5, power_bounced)
+	var/power_bounced = power / 2
```
Comment (RemieRichards) [inline code/game/objects/objs.dm]: * 0.5

Comment (Iamgoofball) [inline code/game/objects/objs.dm]: heresy

Comment (RemieRichards) [inline code/game/objects/objs.dm]: No, speed.

Comment (Iamgoofball) [inline code/game/objects/objs.dm]: speed = heresy

**Comment by Bawhoppen** [comment]
why can't tesla dissipate yet still

**Comment by Aranclanos** [comment]
after the freeze

**Comment by Aranclanos** [comment]
also stop getting distracted and fix objectives goof

**Comment by rev-benji** [comment]
how is this a feature

**Comment by CPTANT** [comment]
Closing this is bullshit. Tesla being way too powerful is an ISSUE.

An issue that won't get fixed if everyone shouts FEATURE FREEZE at everything that is not a spelling correction.

**Comment by Bawhoppen** [comment]
ye this is actually kinda important
also feature freeze has already been violated like 5 times so it's only a formality

**Comment by duncathan** [comment]
@Bawhoppen what the flying fuck are you talking about; no it hasn't
Please answer me this time you cuck

**Comment by Bawhoppen** [comment]
mso's, anturks, someone else's who I forget, probably others

**Comment by duncathan** [comment]
>124 merged PRs in the last week

Yeah no either you give me a link or you shut up about these supposed violations @Bawhoppen

**Comment by Bawhoppen** [comment]
MSO's byond hub one, anturk's jump one, I don't remember others but there probably is

**Comment by Iamgoofball** [comment]
>MSO's byond hub one

not a feature, that's a BYOND setting thing

>anturk's jump one

also not a feature, it's an oversight fix

**Pull request merged**